tests/tracking/test_fluent.py

```diff
@@ -397,3 +398,61 @@ def test_start_run_existing_run_deleted(empty_active_run_stack):
     with mock.patch.object(MlflowClient, "get_run", return_value=mock_run):
         with pytest.raises(MlflowException):
             start_run(run_id)
+
+
+@mock.patch("time.time", lambda: -1)
+@pytest.mark.parametrize("fluent_fn,fluent_kwargs,expected_store_kwargs", [
+    (mlflow.log_metrics, {"metrics": {"a": 0.1}},
+     {"run_id": "my-uuid", "metrics": [Metric("a", 0.1, -1)], "params": [], "tags": []}),
+    (mlflow.log_params, {"params": {"b": "c"}},
+     {"run_id": "my-uuid", "metrics": [], "params": [Param("b", "c")], "tags": []}),
+    (mlflow.set_tags, {"tags": {"d": "e"}},
+     {"run_id": "my-uuid", "metrics": [], "params": [], "tags": [RunTag("d", "e")]}),
+])
+def test_log_batch_apis_delegate_to_store(
```

Comment:
This test and `test_log_batch_client_apis_delegate_to_store` violate the "test the interface not the implementation" principle, but the alternatives would be having no such tests or duplicating all the `store.log_batch` tests for the fluent/client API, which seems excessive (better to just know that the fluent/client APIs correctly call `store.log_batch` and trusting the stores' unit test coverage, especially since it seems unlikely for the fluent/client APIs to undergo a major refactor that'd require deleting/majorly reworking these tests).

Reply:
These tests don't seem to cover any function/input cases that aren't already covered by `test_log_batch`, `test_log_metrics`, `test_set_tags`, or `test_log_params`. We also don't verify that existing fluent/client `log_...()` calls invoke appropriate methods in the `AbstractStore`. Is this test necessary?

Reply:
Yeah it's a good point that we kinda cover this behavior w/ the other tests (like `test_log_metrics` etc) anyways, happy to delete it :P (and the client API test too)